spec/migrations/20180810144738_update_default_internal_attribute_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_migration
+
+describe UpdateDefaultInternalAttribute do
+  let(:service_template_stub) { migration_stub(:ServiceTemplate) }
+
+  migration_context :up do
+    it "sets internal to false when type is not ServiceTemplateTransformationPlan" do
+      st = service_template_stub.create!(:type => 'OtherServiceTemplateTransformationPlan')
+
+      migrate
+
+      expect(st.reload.internal).to be_falsey
```

Comment:
This is wrong, because without your PR, this test would pass anyway (because nil is falsey).  This should say `.to be false` to show you want the literal `false` value